examples/client.rs

```diff
@@ -8,6 +8,7 @@ use std::sync::Arc;
 use argh::FromArgs;
 use tokio::io::{copy, split, stdin as tokio_stdin, stdout as tokio_stdout, AsyncWriteExt};
 use tokio::net::TcpStream;
+use tokio_rustls::rustls;
```

Comment:
Let's merge these with the lines below? `use tokio_rustls::{rustls, TlsConnector}`.